packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -60,6 +60,7 @@ class CupertinoSwitch extends StatefulWidget {
     @required this.value,
     @required this.onChanged,
     this.activeColor,
+    this.trackColor,
```

Comment:
`CupertinoColors.secondarySystemFill` is a const value so you should be able to specify it here.

Reply:
I just noticed `activeColor` is doing the same :sweat_smile:. You could ignore the above suggestion or change `activeColor` to follow the same pattern.

Reply:
I changed it to the default value. But also for the `activeColor` so this is consistent.